islandora.api.php

```diff
@@ -602,7 +608,7 @@ function hook_cmodel_pid_islandora_object_access($op, $object, $user) {
  *   about the outcome. Can also be an array containing multiple
  *   TRUE/FALSE/NULLs, due to how hooks work.
  */
-function hook_islandora_datastream_access($op, $object, $user) {
+function hook_islandora_datastream_access($op, AbstractObject $object, $user) {
```

Comment:
Should reference `AbstractDatastream`, methinks.

Reply:
yes ty!